### PR TITLE
Create supervisord config files from hiera data

### PR DESCRIPTION
**Pull request by iuriaranda**

### CODE REVIEW

File: manifests/config/file.pp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  $user             => undef,
+  $process_name     => '%(program_name)s_%(process_num)s',
+  $startsecs        => 10,
+  $numprocs         => 1,
+  $autostart        => 'true',
+  $autorestart      => 'true',
+  $redirect_stderr  => 'true',
+  $startretries     => 3,
+) {
+  file { "${::supervisord::params::includedir}/${title}.conf":
```
Comment (SamClinckspoor) [inline manifests/config/file.pp:15]: don't you lose the ability to wildcard some files in `/var/www/$sitename/..` here?

Comment (SamClinckspoor) [reply manifests/config/file.pp:15]: for some customers we just wildcard all conf files in a dir and let them manage it.

**Review by SamClinckspoor — APPROVED** [review]

File: manifests/config/file.pp (hunk 1 above)
Comment (iuriaranda) [reply manifests/config/file.pp:15]: ah, I see what you mean now. In that case, I think the files defined through puppet won't work, as the parameter `${::supervisord::params::includedir}` will be always the default supervisord folder.

Comment (iuriaranda) [reply manifests/config/file.pp:15]: fixed in the last commit

**Review by simonrondelez — APPROVED** [review]

**Pull request merged**